.npmignore

```diff
@@ -9,3 +9,5 @@ bower.json
 gulpfile.js
 index.html
 screenshot.png
+gen-tsd.json
+magi-p3-post.js
```

Comment:
This repo uses blacklisting with `.npmignore` instead of whitelisting with `files` property in `package.json`

Reply:
Please add `files` section instead, to align with other components.

Reply:
Done